client/dashboard/me/mcp/setup/index.tsx

```diff
@@ -0,0 +1,437 @@
+					</>
+				) }
+			</PageLayout>
+		);
+	}
+
+	return (
+		<PageLayout size="small" header={ <PageHeader title={ __( 'MCP Client Setup' ) } /> }>
+			<ComponentViewTracker eventName="calypso_dashboard_mcp_setup_view" />
+			{ ! isLoadingUserSettings && (
+				<>
+					<Card>
+						<CardBody>
+							<VStack spacing={ 4 }>
+								<Text as="h1" size="large" weight={ 500 }>
+									{ __( 'Connect AI Assistant to WordPress.com (MCP)' ) }
+								</Text>
+								<Text as="p" size="medium" variant="muted">
+									{ __(
+										'WordPress.com provides MCP (Model Context Protocol) support, which allows AI assistants to interact directly with your WordPress.com account.'
+									) }
+								</Text>
+								<Text as="p" size="medium" variant="muted">
+									{ __(
+										'The JSON configuration below sets up a secure connection between your AI assistant and your WordPress.com account. It works by:'
+									) }
+								</Text>
+								<VStack spacing={ 2 }>
+									<ul style={ { color: '#757575', paddingLeft: '20px', margin: '0' } }>
+										<li>
+											<Text as="p" size="medium" variant="muted">
+												{ __(
+													'Running a bridge server using the WordPress.com-specific MCP package'
+												) }
+											</Text>
+										</li>
+										<li>
+											<Text as="p" size="medium" variant="muted">
+												{ __(
+													'Handling OAuth 2.1 authentication to securely connect to your WordPress.com account'
+												) }
+											</Text>
+										</li>
+										<li>
+											<Text as="p" size="medium" variant="muted">
+												{ __(
+													"Providing real-time access to your account's content and management features"
+												) }
+											</Text>
+										</li>
+									</ul>
+								</VStack>
+							</VStack>
+						</CardBody>
+					</Card>
+
+					<div style={ { marginTop: '24px' } }>
+						<Card>
+							<CardBody>
+								<VStack spacing={ 4 }>
+									<Text as="h1" size="large" weight={ 500 }>
+										{ __( 'MCP Client Configuration' ) }
+									</Text>
+									<SelectControl
+										__next40pxDefaultSize
+										__nextHasNoMarginBottom
+										label={ __( 'MCP Client' ) }
+										value={ selectedMcpClient }
+										options={ mcpClientOptions }
+										onChange={ setSelectedMcpClient }
+										help={ __( 'Choose your MCP client to get the correct configuration format.' ) }
+									/>
+								</VStack>
+							</CardBody>
+						</Card>
+					</div>
+
+					{ ( selectedMcpClient === 'claude' || selectedMcpClient === 'cursor' ) && (
+						<div style={ { marginTop: '24px' } }>
+							<Card>
+								<CardBody>
+									{ /* Quick Setup for Claude Desktop */ }
+									{ selectedMcpClient === 'claude' && (
+										<VStack spacing={ 4 }>
+											<Text as="h1" size="large" weight={ 500 }>
+												{ __( 'Quick Setup' ) }
+											</Text>
+											<Text as="p" size="medium" variant="muted">
```

Comment:
Figma shows the default font size of the Text component so we don't need to specify the size here and below